monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -381,37 +384,63 @@ def get_machine_aws_instance_id(hostname):
     def get_manual_monkey_hostnames():
         return [monkey["hostname"] for monkey in get_manual_monkeys()]
 
-    @staticmethod
-    def get_config_users():
-        return ConfigService.get_config_value(USER_LIST_PATH, True)
+    @classmethod
+    def get_config_users(cls):
```

Comment:
This is unneeded. The UI can take what it needs from credentials (configured/stolen/all users)

Reply:
@ilija-lazoroski already has this logic in the UI

Reply:
It's still needed because of `get_issue_set()` https://github.com/guardicore/monkey/blob/57b97d09f40671b8c455ab5d1d65d889548a481c/monkey/monkey_island/cc/services/reporting/report.py#L500.